utils/backup/backup

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+user="default"
+path="."
+
+usage() {
+    echo
+    echo "A trivial script to upload your files into ClickHouse."
+    echo "You might want to use something like Dropbox instead, but..."
+    echo
+    echo "Usage: $0 --host <hostname> [--user <username>] --password <password> <path>"
+    exit 1
+}
+
+while [[ "$#" -gt 0 ]]; do
+    case "$1" in
+        --host)
+            host="$2"
+            shift 2
+            ;;
+        --user)
```

Comment:
Should table also be configurable?

Reply:
Probably yes, but so far, there is only a single use-case for this tool (I want to backup my files), so no need for prematurly adding features.

Reply:
Understandable.